src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Core/Models/ErrorId.cs

```diff
@@ -54,5 +54,6 @@ public enum ErrorId
     ManagePermissions_Delete_MissingSelection,
     AccountAdmin_TypeOfRole_MissingSelection,
     AccountAdmin_TypeOfUserLa_MissingSelection,
-    AccountAdmin_TypeOfUserVcs_MissingSelection
+    AccountAdmin_TypeOfUserVcs_MissingSelection,
+    Delete_Service__NeitherRadioButtonIsSelected
```

Comment:
nitpick: should this be a double _ or a single?

Reply:
The other ones in the file are all double __ so just followed the convention in there

Reply:
Well some are, looks like the style the previous person used was vibe dependent